kmpauth-google/src/androidMain/kotlin/com/mmk/kmpauth/google/GoogleAuthUiProviderImpl.kt

```diff
@@ -70,15 +89,20 @@ internal class GoogleAuthUiProviderImpl(
         }
     }
 
-    private fun getCredentialRequest(): GetCredentialRequest {
+    private fun getCredentialRequest(checkPreviouslyUsedAccounts: Boolean): GetCredentialRequest {
         return GetCredentialRequest.Builder()
-            .addCredentialOption(getGoogleIdOption(serverClientId = credentials.serverId))
+            .addCredentialOption(
+                getGoogleIdOption(
+                    serverClientId = credentials.serverId,
+                    checkPreviouslyUsedAccounts = checkPreviouslyUsedAccounts
+                )
+            )
             .build()
     }
 
-    private fun getGoogleIdOption(serverClientId: String): GetGoogleIdOption {
+    private fun getGoogleIdOption(serverClientId: String, checkPreviouslyUsedAccounts: Boolean): GetGoogleIdOption {
```

Comment:
pls, change variable name to `filterByAuthorizedAccounts` in order not to create confusion

Reply:
I agree! I changed it to `filterByAuthorizedAccounts` :)